update-work

SondraE opened the pull request:
![](https://images.unsplash.com/photo-1528835903908-c3c61ea16729?crop=entropy&cs=tinysrgb&fit=max&fm=jpg&ixid=MnwxfDB8MXxyYW5kb218MHx8Y3V0ZSxhbmltYWwsQ0hBTkdFX018fHx8fHwxNzExMDYyMzIy&ixlib=rb-4.0.3&q=80&utm_campaign=api-credit&utm_medium=referral&utm_source=unsplash_source&w=1080)
<!-- Swap `CHANGE_ME` for a random string to get a different image -->



## Description
Adding missing clients to the Work list

## Related Issue(s)

- Fixes #539 

## Steps to test/reproduce
- /work/
- Sass Playground & Website
- Estimating Engine

jgerigmeyer commented on content/work/sass.md:
If this is Sass Website only, probably update the filename to `sass-website.md` or something?

jgerigmeyer reviewed (commented):
Can we feature some of these new clients on the home page?

mirisuzanne commented on content/tools/blend.md:
```diff
@@ -25,6 +26,17 @@ summary: |
+We've been helping to
+[design and implement](https://github.com/sass/sass/blob/main/proposal/color-4-new-spaces.md)
+wide-gamut color support
+directly in the [Sass language](/sass/).
```
I think we link to `/sass/` unless we're talking about something we built for them as a client? I don't know…

mirisuzanne replied:
oh nooooooo. I don't have a good answer. Should we try and figure out a way to merge the two?

jgerigmeyer replied:
I'd be curious what @SondraE thinks. My initial leaning might be to leave both in place, but prefer linking to `/work/sass-core/` -- and could even mention on that page that you've been a core contributor since 2016.

SondraE replied:
@mirisuzanne I share Jonny's leaning. And I do think that page should indicate that you've been a core contributor.

SondraE commented:
@mirisuzanne 

> Can we feature some of these new clients on the home page?

I think we should either add another set of four logos to the featured clients section (Chrome, Google, Sass, GMS?) or if that doesn't quite make sense, then I think SRAM and Phamaly would be good candidates to replace. SRAM because it was a long time ago and Phamaly because we aren't looking for more WP work.

jgerigmeyer commented on content/blog/index.njk:
```diff
@@ -6,7 +6,7 @@ action:
   url: /tags/
 pagination:
-  data: collections._post
+  data: collections.posts
```
@mirisuzanne Because we use a custom sort (by `end` date), this needs to be sorted before pagination -- without this, the TimeDesigner case study was paginated with 2017 posts, but displayed as a 2022 post.

jgerigmeyer commented on content/work/sass-site.md:
```diff
@@ -68,7 +82,7 @@ summary: |
 
 - Figma
-- JavaScript with React & Redux
+- JavaScript
```
Do we want to mention Eleventy here too?

SondraE commented:
@mirisuzanne Well...I tried to add Google's logo to the homepage. On index.njk, I replaced `/work/phamaly/` with   `/work/google/`. We already have an svg of google's logo. On the UI Fund page I added `logo: google`. But apparently there's something else I need to change, and I'm not sure what it is.

jgerigmeyer commented:
@SondraE `/work/google/` doesn't exist, which is why it broke. I replaced it with `work/ui-fund/`, which is where I assume you wanted to link to?

I also re-ordered the logos a bit, but feel free to replace that if you like.

mirisuzanne commented:
@SondraE do we need to wait for quotes, or is this ready for final review?

SondraE commented:
@mirisuzanne I don't think we have a way to publish work without a quote at this point, right? I followed up with Ryan this morning, and he said he doesn't have a quote for us yet, but he's planning to get something together in the next few days. I think we're waiting for that.

mirisuzanne commented:
@SondraE fine to wait if we want - but we did change it so we only need a quote OR an image to publish it on the work page.

jgerigmeyer commented on content/work/ui-fund.md:
```diff
@@ -1,21 +1,127 @@
+    name: Rich Harris
+    face:  rich-harris.jpg
+    title: Creator and project lead
+    venue: Svelte JS
```
Do we want to add URLs for either of these people?

mirisuzanne replied:
We could. Links to the tweets, or links to the projects?

jgerigmeyer replied:
Either way -- what have we done elsewhere? It looks to me like we're mostly linking to projects/linkedin.

mirisuzanne commented:
@jgerigmeyer I added urls for those quotes

@SondraE I added Nicole's quotes to a few of the pages. I also added a speaking quote to the home page. See what you think.

jgerigmeyer commented on content/work/ui-fund.md:
```diff
@@ -0,0 +1,169 @@
+  - Container Queries
+  - CSS Scope
+summary: |
+  We were [sponsored](/tools/#open-source-sponsors)
```
Now that we added sponsors to this page, should we just link internally?

```suggestion
  We were [sponsored](#open-source-sponsors)
```

jgerigmeyer commented on content/work/sass-core.md:
```diff
@@ -0,0 +1,95 @@
+    url: https://github.com/nex3
+    venue: *client
+press:
+  - text: |
+      OddBird's team helped power out
+      several new Sass features
+      that would have taken the core team alone
+      years longer.
+      **They dropped in like a task force**,
+      learned the ropes of programming language
+      design and development,
+      and made substantial improvements to the product.
+      We'd love to work with them again!
+    <<: *natalie
```
Do we want to show this quote on this page?

jgerigmeyer replied:
Added in https://github.com/oddbird/oddleventy/pull/540/commits/259a7e0ec3a14b8a844a8bcf441e65929334c96e

SondraE commented:
@mirisuzanne 
This is all looking really good to me. 

The only thing that looks a little strange to me is the way the Services and Language & Tools lists look when there is no image helping to orient them in space. 
<img width="1440" alt="Screenshot 2024-04-16 at 1 36 28 PM" src="https://github.com/oddbird/oddleventy/assets/8933115/f9de3836-7f8c-4e0b-a3fe-d6a396529d24">


Below are the two layouts for those columns on the other projects:
<img width="1440" alt="Screenshot 2024-04-16 at 1 34 12 PM" src="https://github.com/oddbird/oddleventy/assets/8933115/5e97ec7c-e783-488d-ad54-84009ef0ed51">

<img width="1440" alt="Screenshot 2024-04-16 at 1 38 45 PM" src="https://github.com/oddbird/oddleventy/assets/8933115/3b6e8482-e08b-4a7a-9fb8-03e3489c8460">

Could we either add images to the new projects just above those lists so that the pattern is consistent across projects or, if there are no relevant images to add, then could we just stack those lists instead of using the two column layout?

mirisuzanne commented:
@SondraE How's that look? (I also added a Nicole quote to the planning services page)

SondraE commented:
@mirisuzanne Ship it!
